cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,27 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignUpPageObject from '../support/pages/signUp.pageObject';
+
+const signUpPage = new SignUpPageObject();
+
 describe('User', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
-
+  it('should be able to follow the another user', () => {
+    signUpPage.visit();
+    signUpPage.typeUsername(user.username);
+    signUpPage.typeEmail(user.email);
+    signUpPage.typePassword(user.password);
+    signUpPage.clickSignUpBtn();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(1000);
+    cy.visit('#/@riot');
+    cy.get('class="btn btn-sm btn-outline-secondary action-btn"').click();
```

Comment:
The selector `cy.get('class="btn btn-sm btn-outline-secondary action-btn"')` is incorrect. The `cy.get()` method expects a CSS selector, so you should use `.btn.btn-sm.btn-outline-secondary.action-btn` instead. Ensure that the selector correctly targets the follow button.